src/main/java/ui/BoardAutoCreator.java

```diff
@@ -23,6 +23,20 @@
     private static final String MILESTONES = "Milestones";
     private static final String WORK_ALLOCATION = "Work Allocation";
     private static final int MAX_WORK_ALLOCATION_PANELS = 5;
+    public static final String SAMPLE_BOARD = "Sample Board!";
+    private static final String FIRST_SAMPLE_REPO_NAME = "HubTurbo/SampleRepo";
+    private static final String SECOND_SAMPLE_REPO_NAME = "HubTurbo/SampleRepo2";
+
+    public static final String FIRST_SAMPLE_PANEL_NAME = "All issues in my two sample repos";
+    public static final String SECOND_SAMPLE_PANEL_NAME = "Latest 3 urgent open issues";
+    public static final String THIRD_SAMPLE_PANEL_NAME = "Open issues assigned to Darius or Manmeet";
+
+    public static final String FIRST_SAMPLE_FILTER = "(repo:" + FIRST_SAMPLE_REPO_NAME + " | " +
+            "repo:" + SECOND_SAMPLE_REPO_NAME + ")" + " " + "is: issue sort:comments,updated";
+    public static final String SECOND_SAMPLE_FILTER = "repo:" + FIRST_SAMPLE_REPO_NAME + " " +
+            "count:3 is:issue is:open label:\"urgent\"";
+    public static final String THIRD_SAMPLE_FILTER = "repo:" + FIRST_SAMPLE_REPO_NAME
```

Comment:
Could we use an array to store these? Then we don't have to copy-paste for all the rest of the code. Also, format strings might be neater.